proposals/0158-cloud-app-transport-adapter.md

```diff
@@ -115,7 +115,9 @@ Add new RPCs `SetCloudAppProperties` and `GetCloudAppProperties`.
     </enum>
 
     <struct name="CloudAppProperties" since="5.x">
-        <param name="appName" type="String" maxlength="100" mandatory="true"></param>
+        <param name="appName" type="String" minlength="0" maxlength="100" array="true" minsize="0" maxsize="100" mandatory="false">
+            <description>An array of app names a cloud app is allowed to register with. If included in a setCloudAppProperties requests, the array of app names will overwrite the existing nicknames array in the app policies section of the policy table.</description>
```

Comment:
```
... included in a SetCloudAppProperties request, ...
```

Reply:
Change `nicknames array` to`"nicknames" array`